### ISSUE TITLE
Bell pairs generation time does not seem to follow theoretical calculation

### ISSUE THREAD

**Issue by Naphann** [sfc-aqua/quisp#528]
**Describe the bug**

When running a link simulation, entanglement generation without purification or entanglement swap, the time to generate Bell pairs is quicker than the theoretical values. This means that either the loss probability is incorrect or the delay in the communication channel is set incorrectly. 

**To Reproduce**

The theoretical time to reach a certain number of Bell pairs for MIM link can be described by
```python
def calculate_time(L, R, N_pair, N_memory):
    print(f"L: {L}, R: {R}")
    p_bsa = 0.5
    p_loss = 0.0450074
    c = 208189
    t_attemp = 2 * max(L, R) / c
    p_entangle = (1 - p_loss) ** (L+R) * p_bsa
    T = N_pair * t_attemp / p_entangle / N_memory
    return T
```
But the numbers seen from QuISP simulation are less than the theoretical ones.
